pkg/live/rgpath.go

```diff
@@ -35,34 +33,26 @@ func (r *ResourceGroupPathManifestReader) Read() ([]*unstructured.Unstructured,
 	if err != nil {
 		return nil, err
 	}
-	p, err := pkg.New(filesys.FileSystemOrOnDisk{}, absPkgPath)
-	if err != nil {
-		return nil, err
-	}
-
-	// Lookup all files referenced by all subpackages.
-	fcPaths, err := pkg.FunctionConfigFilePaths(filesys.FileSystemOrOnDisk{}, p.UniquePath, true)
-	if err != nil {
-		return nil, err
-	}
 
 	var objs []*unstructured.Unstructured
 	nodes, err := (&kio.LocalPackageReader{
-		PackagePath:     r.PkgPath,
+		PackagePath:     absPkgPath,
 		WrapBareSeqNode: true,
 	}).Read()
 	if err != nil {
 		return objs, err
 	}
 
 	for _, n := range nodes {
-		relPath, _, err := kioutil.GetFileAnnotations(n)
-		if err != nil {
-			return objs, err
-		}
-		if fcPaths.Has(relPath) && !isExplicitNotLocalConfig(n) {
-			continue
-		}
+		// Note(droot): Since we stopped givign special treatment to functionConfigs
```

Comment:
clean up the code?

Reply:
Done. Moved the comment to an appropriate location.